src/derivative_operators/convolutions.jl

```diff
@@ -28,12 +36,27 @@ end
 function convolve_BC_left!(x_temp::AbstractVector{T}, x::AbstractVector{T}, A::DerivativeOperator) where {T<:Real}
     stencil = A.low_boundary_coefs
     coeff   = A.coefficients
-    for i in 1 : A.boundary_point_count
-        cur_stencil = stencil[i]
+
+    _bpc = A.boundary_point_count
+    flag = false
```

Comment:
what exactly does this flag do? Name it.

Reply:
Ok. This is when there are no boundary points.